pyccel/parser/semantic.py

```diff
@@ -3388,6 +3396,10 @@ def _visit_FunctionDef(self, expr):
             self._allocs.append([])
 
             # we annotate the body
+            if cls_name and expr.name == '__Pyccel__del__':
```

Comment:
Where does the `__Pyccel` come from? Do we have 2 freeing functions?

Reply:
Fixed f31858d2e5402008754c6cf25de3381f8c0bca94.